src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+##########################
+#    Overworld Blocks    #
+##########################
+
+tile.natura.clouds.white.name=Wolke
+tile.natura.clouds.white.tooltip=So leicht und flauschig!\nRettet Dich, wenn Du fällst
+tile.natura.clouds.dark.name=Dunkle Wolke
+tile.natura.clouds.dark.tooltip=So leicht und flauschig!\nVerursacht Blitzeinschläge
+tile.natura.clouds.ash.name=Aschewolke
+tile.natura.clouds.ash.tooltip=So leicht und flauschig!\n3x3 Verwandelt sich in Holzkohle
+tile.natura.clouds.sulfur.name=Schwefelwolke
+tile.natura.clouds.sulfur.tooltip=So leicht und flauschig!\n2x2 Verwandelt sich in Schwefel
+
+tile.natura.overworld_logs.maple.name=Ahornholz
+tile.natura.overworld_logs.maple.tooltip=Etwas süßlich
+tile.natura.overworld_logs.silverbell.name=Silberglöckchenholz
+tile.natura.overworld_logs.silverbell.tooltip=Silberne Glocken
+tile.natura.overworld_logs.amaranth.name=Amaranthholz
+tile.natura.overworld_logs.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_logs.tiger.name=Tigerholz
+tile.natura.overworld_logs.tiger.tooltip=Wildgetreide
+
+tile.natura.overworld_logs2.willow.name=Weidenholz
+tile.natura.overworld_logs2.willow.tooltip=Der Weinende
+tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Sakuraholz
+tile.natura.overworld_logs2.sakura.tooltip=Blühende Kirsche
+tile.natura.overworld_logs2.hopseed.name=Hopfensaatholz
+tile.natura.overworld_logs2.hopseed.tooltip=Aufgestiegener Fehler
+
+tile.natura.overworld_leaves.maple.name=Ahornblätter
+tile.natura.overworld_leaves.maple.tooltip=Etwas süßlich
+tile.natura.overworld_leaves.silverbell.name=Glockenblumenblätter
+tile.natura.overworld_leaves.silverbell.tooltip=Silberne Glocken
+tile.natura.overworld_leaves.amaranth.name=Amaranthblätter
+tile.natura.overworld_leaves.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_leaves.tiger.name=Tigerblätter
```

Comment:
This should be "Tigerholzblätter" since the name of the tree is "Tigerholz".

Reply:
Well, I also though about that, but we say e.g. "Eichenblätter" and not "Eichenholzblätter".
That's way I went with this solution. O.o

Reply:
"Eichenblätter" comes from the plant called "Eiche". The plant here is called "Tigerholz" so this has to be called "Tigerholzblätter".

Reply:
OK, I'll correct it.